### DISCUSSION TITLE
How to connect to trino with oauth2 authentication

### DISCUSSION THREAD

**Discussion by shawnzhu** [dbeaver/dbeaver#12763] [Q&A]
I got the following error when opening a Trino connection configured with connection property `externalAuthentication=true`:

```
Desktop Browser is not available. Make sure your Java process is not in headless mode (-Djava.awt.headless=false)
```

Background: It uses OAuth2 as authentication type for this Trino server. the authentication itself works when accessing its web UI.

May I ask how to setup `java.awt.headless=false` when connecting to trino? just leave it into the driver properties?

**Comment by musicmuthu** [comment]
any helping hands
